2-api/key-concepts.md

```diff
@@ -2,64 +2,77 @@
 
 # Key concepts
 
-To best understand the reference documentation, you must first know these key concepts.
+To best understand the reference documentation, you must first understand these
+key concepts.
 
 ## Attribute
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** or **model attribute** is a comment label used for review.
```

Comment:
I don't love this definition but don't have a better suggestion at the moment. Is there language we can pull from elsewhere in the documentation that is more clear?